docker/ds/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM openjdk:11-jdk 
+
+# Use this to directly add the opendj zip file
+COPY opendj.zip /var/tmp/
+
+WORKDIR /var/tmp/bootstrap
+
+COPY bootstrap/ /var/tmp/bootstrap/
+
+# The default BASE_DN of the directory tree.
+ENV BASE_DN o=userstore
+
+RUN ./setup.sh
+
+#####################  Runtime container ################
+FROM openjdk:11-jdk 
+
+COPY --from=0 /var/tmp/bootstrap/run/dsrs1 /opt/opendj/
+
+RUN useradd -d /opt/opendj -G root -m -u 11111 forgerock \
+    && apt-get update && apt-get install -y --no-install-recommends dumb-init vim
+   
+WORKDIR /opt/opendj 
```

Comment:
Should the directory also be moved over to /opt/ds instead of /opt/opendj ?

Reply:
The opendj usage is fairly pervasive throughout the products / scripts. I'm a little hesitant to change it everywhere.